babel.config.js

```diff
@@ -3,18 +3,15 @@ module.exports = {
         [
             '@babel/preset-env',
             {
-                useBuiltIns: 'usage',
```

Comment:
@Ofigelov I think we still need `useBuiltIns: 'entry'` to be added

Reply:
done